src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/openapi/ServiceDescriptionsApiControllerIntegrationTest.java

```diff
@@ -39,7 +39,7 @@
 import org.niis.xroad.restapi.openapi.model.ServiceDescriptionDisabledNotice;
```

Comment:
I kind of get the feeling that more tests for refreshOpenApi3Description could be good (here and in ServiceDescriptionServiceIntegrationTest)? Now `ServiceDescriptionsApiControllerIntegrationTest` tests only WSDL related operations?

Maybe "ignore warnings" feature would be good to test in ServiceDescriptionServiceIntegrationTest? Especially if we would add "add/remove services" warnings.

Reply:
Yeah, I guess I could write some more tests. The reason for writing just one test for making sure refreshing works correctly is that these test would be near to identical to those for testing updating service description. Update is tested more carefully and I felt writing the same tests twice wouldn't be necessary. But it's true the implementations aren't identical and I'll write some tests.

Reply:
Added a couple of tests.